app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/destination/entervillainfo/EnterVillaInformationDestination.kt

```diff
@@ -173,12 +174,14 @@ private fun ChangeAddressEnterVillaInformationScreen(
       ),
     )
     Spacer(modifier = Modifier.height(8.dp))
-    ChangeAddressSwitch(
-      label = stringResource(id = R.string.CHANGE_ADDRESS_SUBLET_LABEL),
-      checked = uiState.isSublet.input,
-      onCheckedChange = onIsSubletSelected,
-      onClick = { onIsSubletSelected(!uiState.isSublet.input) },
-    )
+    Row(modifier = Modifier.padding(horizontal = 16.dp)) {
+      ChangeAddressSwitch(
+        label = stringResource(id = R.string.CHANGE_ADDRESS_SUBLET_LABEL),
+        checked = uiState.isSublet.input,
+        onCheckedChange = onIsSubletSelected,
+        onClick = { onIsSubletSelected(!uiState.isSublet.input) },
+      )
+    }
```

Comment:
Since this only has one item inside the row, could the change here just be that we add the padding modifier on the `ChangeAddressSwitch` and we do not introduce a wrapping Row?